src/modules/app/StateType.js

```diff
@@ -24,6 +25,7 @@ export type CategoryRouteStateType = {|
 |}
 
 export type EventRouteStateType = {|
+  +success: true,
```

Comment:
I think this is not needed because you can if there is no `error` then it is a success.

Reply:
Yes, it is required. See: https://flow.org/en/docs/types/unions/#toc-disjoint-unions-with-exact-types